docs/2.8.0/docs/tools/assistant-build.rst

```diff
@@ -15,21 +15,502 @@ To compile your Daml source code into a Daml archive (a ``.dar`` file), run::
 
 You can control the build by changing your project's ``daml.yaml``:
 
-``sdk-version``
-  The SDK version to use for building the project.
-
 ``name``
   The name of the project.
 
+``version``
+  The project version.
+
 ``source``
   The path to the source code.
 
-The generated ``.dar`` file is created in ``.daml/dist/${name}.dar`` by default. To override the default location, pass the ``-o`` argument to ``daml build``::
+By default, the generated ``.dar`` file is created in ``.daml/dist/${name}-${version}.dar``. To override the default location, pass the ``-o`` argument to ``daml build``::
 
   daml build -o path/to/darfile.dar
 
 
 Build multiple packages
 ***********************
 
-*New multi-package build content goes here!*
+``daml build`` supports multi-package builds, an optional feature for building
+and managing multiple interdependent packages simultaneously. The multi-package
+build feature consists of a configuration file and a set of flags.
+
+You can use multi-package builds to:
+
+-  Configure ``daml build`` to automatically rebuild DARs used in data
+   dependencies, when the source code corresponding to those DARs
+   changes
+-  Build all of the packages in a project simultaneously, using ``daml build --all``
+-  Clean all build artifacts in a project, using ``daml clean --all``
+
+Overview
+========
+
+To configure which packages to build automatically, add the configuration file ``multi-package.yaml`` at the root of your project:
+
+``multi-package.yaml``
+
+.. code:: yaml
+
+   packages:
+   - <path to first package>
+   - <path to second package>
+
+You can then run ``daml build --all`` from the root of your
+project to build all the packages listed in the ``multi-package.yaml`` file. Or you can run ``daml build`` within a package to build that package and its dependencies.
+
+The multi-package feature of ``daml build`` is optional. To use multi-package, 
+create a ``multi-package.yaml`` file that reflects your project structure. 
+You don't need to update ``daml.yaml`` or any other files.
+
+.. note::
+
+    ``multi-package.yaml`` assumes that dependent packages use the default
+    ``daml build`` locations for DAR files. If you have shell scripts that move a
+    package's DAR file after building, you'll need to use the 
+    :ref:`output flag <output-flag>`. 
```

Comment:
Perhaps word this as "replace" with the output flag, rather than "use".

Reply:
Changed to "you'll need to use the output flag instead"... which was your original wording; not sure how I lost it :)